crates/bevy_sprite/src/mesh2d/color_material.wgsl

```diff
@@ -11,8 +11,14 @@ struct ColorMaterial {
     color: vec4<f32>,
     // 'flags' is a bit field indicating various options. u32 is 32 bits so we have up to 32 options.
     flags: u32,
+    alpha_cutoff: f32,
 };
-const COLOR_MATERIAL_FLAGS_TEXTURE_BIT: u32 = 1u;
+
+const COLOR_MATERIAL_FLAGS_TEXTURE_BIT: u32              = 1u;
+const COLOR_MATERIAL_FLAGS_ALPHA_MODE_RESERVED_BITS: u32 = 3221225472u; // (0b11u32 << 30)
+const COLOR_MATERIAL_FLAGS_ALPHA_MODE_OPAQUE: u32        = 0u;          // (0u32 << 30)
+const COLOR_MATERIAL_FLAGS_ALPHA_MODE_MASK: u32          = 1073741824u; // (1u32 << 30)
+const COLOR_MATERIAL_FLAGS_ALPHA_MODE_BLEND: u32         = 2147483648u; // (2u32 << 30)
```

Comment:
I would just write the expressions, constants are guaranteed to be evaluated at compile time (see https://doc.rust-lang.org/reference/const_eval.html#constant-expressions)

Reply:
It's not supported in wgsl unfortunately. We should maybe look into using shader_defs for those though.

Reply:
Oh I didn't even notice this snippet was in a wgsl file, the syntax is too similar :sweat_smile:  Sorry about that.